app/webpacker/styles/colours.scss

```diff
@@ -1,24 +1,25 @@
 $black: #1d1d1b;
-$blue: #2881be;
-$blue-dark: #0076bd;
+$blue: #00b1eb;
+$blue-dark: #1877cf;
 $blue-dark-90: rgba($blue-dark, .9);
-$green: #159964;
-$green-dark: #009157;
-$green-dark-90: rgba($green-dark, .9);
+$cta-green: #00703c;
+$cta-green-dark: #005a30;
+$cta-green-dark-90: rgba($cta-green-dark, .9);
+$green: #00ab92;
 $grey: #f0f0f0;
 $grey-light: #cccccc;
 $grey-dark: #656565;
 $grey-mid: #b7b9bb;
-$pink: #db5e9e;
+$pink: #f162a3;
 $pink-dark: #d34491;
 $pink-dark-90: rgba($pink-dark, .9);
-$purple: #8967aa;
+$purple: #ad52a4;
 $purple-dark: #7d58a0;
 $purple-dark-90: rgba($purple-dark, .9);
 $white: #ffffff;
 $white-90: rgba($white, .9);
 $yellow-light: #fef0cb;
```

Comment:
@garethwilliamsdesign I will raise a **separate** ticket to review the additional colours such as this `$yellow-light` in case there are further changes you want to make in light of the brand colour changes?

But this would be a low priority and one for a future sprint - not now!

Reply:
Sounds good.